### DISCUSSION TITLE
Open Popup window from another Popup page

### DISCUSSION THREAD

**Discussion by IndexOutOfLimit** [xamarin/XamarinCommunityToolkit#1896] [Q&A] [answered]
I have a requirement to open a popup page from another popup page. Just wanted to make sure whether it is possible or not?

**Comment by pictos** [comment, marked as answer]
@IndexOutOfLimit, right now this is not supported on all platforms. If I'm not wrong iOS doesn't allow that but works on Android. And we don't have plans to implement it on this package.
